### ISSUE TITLE
Variable use like "a+1 = ${$a + 1}" isn't reported but silently changed by --fix to "a+1 = ${a} + 1"

### ISSUE THREAD

**Issue by vicinus** [rodjek/puppet-lint#749]
Simple example:

```
$a = 1
$b = "a = ${a}; a+1 = ${$a+1}"
notice($b)
```

With puppet-lint 2.3.0 this get changed with --fix without any message to:

```
$a = 1
$b = "a = ${a}; a+1 = ${a}+1"
notice($b)
```

**Comment by rodjek** [comment]
This is basically the same problem as #703, which has been fixed by #752 and will be part of the next release.

```
asmodean :0: code/puppet-lint (git:master → origin {2} ?:3!)$ bin/puppet-lint -f 749.pp 
asmodean :0: code/puppet-lint (git:master → origin {2} ?:3!)$ cat 749.pp  
$a = 1
$b = "a + ${a}; a+1 = ${a+1}"
notice($b)
```
